Instagram/app/src/main/java/com/example/instagram/fragments/ProfileFragment.java

```diff
@@ -25,19 +25,6 @@
 public class ProfileFragment extends PostsFragment {
     private static final String TAG = "ProfileFragment";
     private SwipeRefreshLayout swipeContainer;
-
```

Comment:
So this is duplicating what the parent class is doing. We start looking for the method in the class itself, and if we don't find it, we look in the parent.

If you do not define 'onViewCreated' in the child, it will look at the parent for the method. So
```
 public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
 } 
```
 is the same as not typing anything at all.

Also, queryPosts is overridden here, so the parent's onViewCreated will use the queryPosts defined here. 

Here's an example you can see
https://runestone.academy/ns/books/published/apcsareview/OOBasics/ooSuper.html